caffe2/python/tutorials/MNIST.ipynb

```diff
@@ -553,6 +553,10 @@
     "workspace.RunNetOnce(train_model.param_init_net)\n",
     "# creating the network\n",
     "workspace.CreateNet(train_model.net, overwrite=True)\n",
+    "# Also initialize the test model before training\n",
+    "workspace.RunNetOnce(test_model.param_init_net)\n",
```

Comment:
@salexspb Thank you. I kinda understand why you ask for the test_model.param_init_net.Proto(). It actually contains 5 ops: `CreateDB`, `conv1_w`, `conv1_b`, `conv2_w`, `conv2_b`. You can find the attachment in above discussion. The expected result should be only `CreateDB`. So I started #824 to fix the root cause.